packages/types/src/extensions/pn-any-stream-reference-based-types.ts

```diff
@@ -6,7 +6,10 @@ export {
 } from './pn-any-reference-based-types';
 
 /** Parameters for the creation action */
-export interface ICreationParameters extends PnReferenceBased.ICreationParameters {
-  expectedFlowRate: string;
-  expectedStartDate: string;
+export interface ICreationParameters extends Partial<PnReferenceBased.ICreationParameters> {
+  expectedFlowRate?: string;
+  expectedStartDate?: string;
+  previousRequestId?: string;
+  masterRequestId?: string;
+  recurrenceNumber?: string;
```

Comment:
Why not make recurrenceNumber a number?
Is there a benefit to be string?

Reply:
I updated it to number, it's clearer this way.